update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml

```diff
@@ -63,47 +63,6 @@
     path: /srv/grafana/grafana.db
   register: sqlite_grafana
 
-- name: Remove old dashboards from SQLite
```

Comment:
This part is seamingly unrelated to DBaaS. I therefore suggest we move it to a separate PR which will have similar fixes or removals.

Reply:
actually it does, one of dashboards are DBaaS related, so I decided to remove the whole block, because it won't be required in PMM 2 > PMM 3 migration